antarest/study/storage/auto_archive_service.py

```diff
@@ -84,6 +85,10 @@ def _try_archive_studies(self) -> None:
                     exc_info=e,
                 )
 
+        self.study_service.storage_service.variant_study_service.clear_all_snapshots(
```

Comment:
I preferred the old way when you called this inside a separate method but it's not that important.

What's more important is that you didn't resolve the comment I put that would simplify the code. I'll reopen it and you can answer there to explain why you didn't do it.

Reply:
I'll improve logs in [this branch](https://github.com/AntaresSimulatorTeam/AntaREST/tree/fix/improve-duration-in-logs-of-snapshot-tasks).